interface/billing/indigent_patients_report.php

```diff
@@ -219,10 +219,10 @@ function bucks($amount)
         $arow = sqlQuery("SELECT SUM(pay_amount) AS pay, " .
           "sum(adj_amount) AS adj FROM ar_activity WHERE " .
           "pid = ? AND encounter = ?", array($patient_id, $encounter_id));
-        $inv_paid   += $arow['pay'];
-        $inv_amount -= $arow['adj'];
-        $total_amount += bucks($inv_amount);
-        $total_paid   += bucks($inv_paid);
+        $inv_paid   += floatval($arow['pay']);
+        $inv_amount -= $floatval(arow['adj']);
+        $total_amount += $inv_amount;
+        $total_paid   += $inv_paid;
```

Comment:
Don't these changes, which not run it through bucks() then lose the internationalization features since bucks() uses oeFormatMoney() function?

Reply:
Notice this is summing and the formatting will happen later at display time a little further down in code. It troubled me to do math on a formatted value.